eng/Versions.props

```diff
@@ -159,7 +159,7 @@
   </PropertyGroup>
   <PropertyGroup Label="Manual">
     <!-- Packages from dotnet/roslyn -->
-    <MicrosoftNetCompilersToolsetVersion>3.10.0-1.final</MicrosoftNetCompilersToolsetVersion>
+    <MicrosoftNetCompilersToolsetVersion>4.0.0-2.21354.7</MicrosoftNetCompilersToolsetVersion>
```

Comment:
This line shouldn't do anything and should be removed from the repo. See https://github.com/dotnet/aspnetcore/pull/33393#discussion_r651107151 and the changes in #33393. IIRC we use the toolset version burned into `msbuild` and `dontet msbuild` now. Builds probably pick up `v4.0.0-2.21321.3` or `v4.0.0-2.21354.7` at the moment. See https://github.com/dotnet/sdk/blob/ef6249b3b034b12f127b0d3a72ea428979f44413/eng/Versions.props#L120

Please instead please remove this confusing line.

Reply:
Got it -- admittedly I was confused on what was relevant and what wasn't and opted to update everything.

Makes sense to remove this alltogether.

Reply:
thanx